scene/resources/gradient_texture.cpp

```diff
@@ -172,14 +172,13 @@ RID GradientTexture1D::get_rid() const {
 }
 
 Ref<Image> GradientTexture1D::get_image() const {
-	const_cast<GradientTexture1D *>(this)->update_now();
```

Comment:
this and the other call to `update_now()` should still be here, just as a regular method call without the `const_cast`